components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/saml/SAML2BearerGrantHandler.java

```diff
@@ -732,7 +737,7 @@ private String getTokenEPAliasFromResidentIdp(Assertion assertion, IdentityProvi
                                                   String tenantDomain) throws IdentityOAuth2Exception {
         String tokenEndpointAlias = null;
         FederatedAuthenticatorConfig[] fedAuthnConfigs = identityProvider.getFederatedAuthenticatorConfigs();
-        validateIdpEntityId(assertion, tenantDomain,  getIdpEntityId(fedAuthnConfigs));
+        //validateIdpEntityId(assertion, tenantDomain,  getIdpEntityId(fedAuthnConfigs));
```

Comment:
Why have we commented this line? Can't we remove it?

Reply:
Addressed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1438